suites/pacific/cephadm/tier-1_remove_cluster.yaml

```diff
@@ -1,9 +1,10 @@
 #===============================================================================================
-# Conf: conf/pacific/cephadm/tier-1.yaml
+# Conf: conf/pacific/cephadm/tier-1_5node_cephadm_bootstrap.yaml
```

Comment:
Why this need to be changed ? Existing TCs will work with new conf file ?

Reply:
There is no conf/pacific/cephadm/tier-1.yaml in the directory!